### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by josephfrazier**
This should get the [build](https://travis-ci.org/arijs/prettier-miscellaneous/branches) working again.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,8 @@
   },
   "scripts": {
     "jest": "jest",
-    "test": "npm run jest && npm run test-tabs && npm run test-tabs-inv",
+    "test": "npm run jest",
+    "posttest": "npm run test-tabs && npm run test-tabs-inv",
```
Comment (rhengles) [inline package.json]: Umm. If this is the case, can't we do this?

```diff
-    "jest": "jest",
-    "test": "npm run jest && npm run test-tabs && npm run test-tabs-inv",
+    "test": "jest",
+    "posttest": "npm run test-tabs && npm run test-tabs-inv",
```

**Comment by rhengles** [comment]
You're awesome!

**Comment by langdonx** [comment]
Haha, awesome, was looking to tackle this myself earlier.

**Comment by josephfrazier** [comment]
Thanks everyone!

**Pull request merged**